draft-ietf-rats-architecture.md

```diff
@@ -828,12 +828,12 @@ In some scenarios, Evidence might contain sensitive information such as
 Personally Identifiable Information.
 Thus, an Attester must trust entities to which it conveys Evidence, to not
 reveal sensitive data to unauthorized parties.
-The Verifier might share this information with other authorized parties, according to rules that it controls.
+The Verifier might share this information with other authorized parties, according to some predefined rules.
 In the background-check model, this Evidence may also be revealed to Relying Party(s).
 
-In some cases where Evidence contains sensitive information, an Attester
-might even require that a Verifier first go through a TLS authentication or a remote attestation procedure with it before the Attester
-will send the sensitive Evidence.  This can be done by having the
+When Evidence contains sensitive information, an Attester
```

Comment:
Seems the intend here was to say "this is not always the case when sensitive information is included, but sometimes".

Also, what is sensitive information in the first place? Does sensitive information require extensive integrity protection, because it breaks easily? Seems a tad bit too colloquial to me - not only here, but in the surrounding text.

Reply:
> Seems the intend here was to say "this is not always the case when sensitive information is included, but sometimes".

I added "typically" below because while this is optional, it makes sense to take some precaution in this specific case.

> Also, what is sensitive information in the first place? Does sensitive information require extensive integrity protection, because it breaks easily? Seems a tad bit too colloquial to me - not only here, but in the surrounding text.

This is in the context of PII handling between mutually trusting parties, which is established by the preceding paragraph.